README.md

```diff
@@ -27,14 +27,14 @@ this is why a build directory is created.
 ```bash
 $ mkdir gccrs-build
 $ cd gccrs-build
-$ ../gccrs/configure --prefix=$HOME/gccrs-install --disable-bootstrap --enable-multilib --enable-languages=c,c++,rust
+$ ../gccrs/configure --prefix=$HOME/gccrs-install --disable-bootstrap --enable-multilib --enable-languages=rust
 $ make
 ```
 
 Running the compiler itself without make install we can simply imvoke the compiler proper:
```

Comment:
typo:
```suggestion
Running the compiler itself without make install we can simply invoke the compiler proper:
```
(you might want to manually commit it as this change is small enough to not require licensing)

Reply:
thanks good find :+1: